src/common/Database/QueryResult.h

```diff
@@ -84,7 +84,7 @@ class PreparedResultSet
         MYSQL_STMT* m_stmt;
         MYSQL_RES* m_res;
 
-        my_bool* m_isNull;
+        bool* m_isNull;
```

Comment:
@Kitzunu try to revert this change + apply this PR:

https://github.com/azerothcore/azerothcore-wotlk/pull/3060